vms/platformvm/txs/executor/standard_tx_executor.go

```diff
@@ -545,3 +528,29 @@ func (e *StandardTxExecutor) BaseTx(tx *txs.BaseTx) error {
 	avax.Produce(e.State, e.Tx.ID(), tx.Outs)
 	return nil
 }
+
+// addStakerFromStakerTx creates the staker and adds it to state.
+func (e *StandardTxExecutor) addStakerFromStakerTx(stakerTx txs.Staker) error {
```

Comment:
nit: wdyt about `putStaker`? `addStakerFromStakerTx` stutters and I'd like to retain the `put` language

Reply:
Done